### PR TITLE
Update teams-scoped-directory-search.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6211
Added link

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-scoped-directory-search.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ Microsoft Teams lets organizations provide custom views of the directory to thei
 > [!NOTE]
 > In Exchange hybrid environments, this feature only works with Exchange Online mailboxes, and not with on-premises mailboxes.
 
+See also [Address book policies in Exchange Online](https://docs.microsoft.com/exchange/address-books/address-book-policies/address-book-policies)
```
Comment (JohanFreelancer9) [inline Teams/teams-scoped-directory-search.md:34]:
```suggestion
See also [Address book policies in Exchange Online](https://docs.microsoft.com/exchange/address-books/address-book-policies/address-book-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by cazawideh — APPROVED** [review]

**Pull request merged**